src/components/Comments.jsx

```diff
@@ -1,24 +1,21 @@
 import { useEffect, useState } from "react";
 import { formatDate, getCommentsByArticleId } from "./utils";
-import { LiaComments } from "react-icons/lia";
-import {
-	BiSolidUpvote,
-	BiChat,
-	BiChevronsDown,
-	BiChevronsUp,
-} from "react-icons/bi";
+import { BiChat, BiChevronsDown, BiChevronsUp } from "react-icons/bi";
 import { Box } from "./Box";
 import {} from "react-icons/bi";
 
 export const Comments = ({ article_id }) => {
 	const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] =useState(false)
```

Comment:
the initial state of this should be true, and just set to false in the then & catch blocks